test/unit/content-src/components/SectionMenu.test.jsx

```diff
@@ -164,17 +164,6 @@ describe("<SectionMenu>", () => {
   });
   describe(".onClick", () => {
     const dispatch = sinon.stub();
-    const propOptions = [
-      "Separator",
-      "MoveUp",
-      "MoveDown",
-      "RemoveSection",
-      "CollapseSection",
-      "ExpandSection",
-      "ManageSection",
-      "AddTopSite",
-      "PrivacyNotice",
-    ];
```

Comment:
Are we removing this because its not used or its breaking test?

Reply:
Removing because its not being used. In section menu we don't have a prop for propOptions but just base the default options on the webextension and call it propOptions in the component

https://github.com/mozilla/activity-stream/blob/0379c988b17bcdef56bd8ac880368b0e636a2035/content-src/components/SectionMenu/SectionMenu.jsx#L29-L34